schedule/yast/maintenance/autoyast_create_hdd_maintenance_nopatch.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name: autoyast_create_hdd_maintenance_nopatch
```

Comment:
For later migration test, we'd better de-registered system before publish qcow. You can use
schedule/yast/maintenance/autoyast_create_hdd_maintenance_deregister.yaml, or update it for your need.

Reply:
The system is already not registered. If we omit the "register" section in the profile, then it's not register, maybe we can try it also for other product, and then remove the de-register module everywhere we use autoyast, if that works. I suspect it would not work for sle15 online medium, but works here.

Reply:
susetest:~ # SUSEConnect --status-text
Installed Products:
------------------------------------------

  SUSE Linux Enterprise Server for SAP Applications 12 SP4
  (SLES_SAP/12.4/x86_64)

  Not Registered

Reply:
I tried to use first the main create_hdd_ha.xml.ep as we discussed, but that was not working, SAP requires different registration codes and partitioning, so it would require a lot of conditions in the profile.

Reply:
In the end, the partitioning and registration could be the same, but it's a different installation medium and requires specific packages to be installed, so I kept the specific profile.